tests/e2e/specs/editor/publishingFlow.js

```diff
@@ -69,6 +69,9 @@ describe('Publishing Flow', () => {
     await expect(page).toClick('a', { text: 'Add to new post' });
     await page.waitForNavigation();
 
+    // See https://github.com/WordPress/gutenberg/blob/c31555d4cec541db929ee5f63b900c6577513272/packages/e2e-test-utils/src/create-new-post.js#L37-L63.
+    await page.waitForSelector('.edit-post-layout');
```

Comment:
I think the issue, was we needed to wait for the gutenberg editor to load fully. Waiting on this selector will do that. Everything else is just extra, but can't hurt anything.